src/styles/blocks/card.scss

```diff
@@ -0,0 +1,114 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+
+  max-width: 200px;
+  margin-right: 48px;
+  padding: 0 16px 16px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(#{$cards-in-a-row}n) {
+    margin-right: 0;
+  }
+
+  &:nth-child(n + #{$cards-in-a-row + 1}) {
+    margin-top: 46px;
+  }
+
+  &__image-container {
+    @extend %flex-centre;
+    margin-top: 32px;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin: 40px 0 0;
+
+    @extend %normal-text;
+    font-weight: 500;
+
+    color: $main-text-color;
+  }
+
+  &__product-code {
+    margin: 4px 0 0;
+
+    @extend %small-text;
+    font-weight: 400;
+
+    color: $secondary-text-color;
+  }
+
+  &__paragraph {
+    margin: 0;
+  }
+
+  &__rate-block {
+    @extend %flex-space-between;
+    align-items: flex-end;
+    margin-top: 16px;
+
+    @extend %small-text;
+  }
+
+  &__rate-block-reviews {
+    margin: 0;
+  }
+
+  &__price-block {
+    @extend %flex-space-between;
+
+    margin: 24px 0 0;
+  }
+
+  &__price-block-paragraph {
+    margin: 0;
+
+    @extend %normal-text;
+
+    color: $secondary-text-color;
+  }
+
+  &__price-block-sum {
+    margin: 0;
+
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+
+    color: $main-text-color;
+  }
+
+  &__buy-button {
+    cursor: pointer;
+    box-sizing: border-box;
```

Comment:
add box-sizing to normalize under `*` selector, it should be the same for all elements
```suggestion
```